tools/clang/test/SemaHLSL/spec.hlsl

```diff
@@ -194,17 +195,17 @@ namespace ns_std_conversions {
     fn_f14(1);
 
     u = f11; // matrix single element conversion
-    u = f14; // matrix scalar truncation conversion
+    u = f14; // matrix scalar truncation conversion // expected-warning {{implicit truncation of vector type}}
```

Comment:
The doubled up comments are gross. Can you clean this up by moving one of the comments to the line above?

Please look for other places where this occurs too.